cishouseholds/pipeline/mapping.py

```diff
@@ -1434,6 +1434,158 @@
+        "PHM-FUP-C19-029": "last_28_days_pharmacist_111_respiratory_infection",
+        "PHM-FUP-C19-030": "last_28_days_urgent_care",
+        "PHM-FUP-C19-031": "last_28_days_urgent_care_respiratory_infection",
+        "PHM-FUP-C19-032": "last_28_days_admitted_to_hospital",
+        "PHM-FUP-C19-033": "last_28_days_admitted_to_hospital",
+        "PHM-FUP-C19-034": "think_have_long_covid",
+        "PHM-FUP-C19-035": "think_have_long_covid_symptom_reduced_ability",
+        "PHM-FUP-C19-036": "think_have_long_covid_symptom_list_1",
+        "PHM-FUP-C19-038": "think_have_long_covid_symptom_list_2",
+        "PHM-FUP-C19-040": "think_have_long_covid_symptom_list_3",
+        # "PHM-FUP-C19-038": "other_antibody_test_results",
+        "PHM-FUP-C19-039": "other_antibody_test_positive_date",
+        # "PHM-FUP-C19-040": "other_antibody_test_negative_date",
+        "PHM-FUP-C19-042": "think_have_long_covid_symptom_worse_after_effort",
+        "PHM-FUP-CON-001": "hospital_last_28_days",
+        "PHM-FUP-CON-002": "care_home_last_28_days",
+        "PHM-FUP-CON-003": "face_covering_other_enclosed_places",
+        "PHM-FUP-CON-004": "physical_contact_under_18_years",
+        "PHM-FUP-CON-005": "physical_contact_18_to_69_years",
+        "PHM-FUP-CON-006": "physical_contact_over_70_years",
+        "PHM-FUP-CON-007": "social_distance_contact_under_18_years",
+        "PHM-FUP-CON-008": "social_distance_contact_18_to_69_years",
+        "PHM-FUP-CON-009": "social_distance_contact_over_70_years",
+        "PHM-FUP-CON-010": "times_hour_or_longer_another_home_last_7_days",
+        "PHM-FUP-CON-011": "times_hour_or_longer_another_person_your_home_last_7_days",
+        "PHM-FUP-CON-012": "times_shopping_last_7_days",
+        "PHM-FUP-CON-013": "times_socialise_last_7_days",
+        "PHM-FUP-CON-014": "times_indoor_exercise_last_28_days",
+        "PHM-FUP-CON-015": "times_large_events_last_28_days",
+        "PHM-FUP-CON-016": "times_medium_events_last_28_days",
+        "PHM-FUP-CON-018": "transport_shared_outside_household_last_28_days_none",
+        "PHM-FUP-CON-019": "face_covering_work_or_education",
+        "PHM-FUP-END-001": "end_screen_questionnaire",
+        "PHM-FUP-END-002": "end_screen_sample",
+        "PARTICIPANT_ID": "participant_id",
+        "FIRST_NAME": "first_name",
+        "BLOOD_TEST_BARCODE": "blood_sample_barcode",
+        "SWAB_TEST_BARCODE": "swab_sample_barcode",
+        "COLLEX_OPEN_DATE": "participant_completion_window_start_date_string",
+        "COLLEX_CLOSE_DATE": "participant_completion_window_end_date_string",
+        "PORTAL_ID": "portal_id",
+        "WINDOW_START_DATE": "participant_completion_window_start_date",
+        "WINDOW_CLOSE_DATE": "participant_completion_window_end_date",
+        "PARTICIPANT_WINDOW_ID": "participant_completion_window_id",
+        "submitted_at": "survey_completed_datetime",
+        "flushed": "survey_completion_status_flushed",
```

Comment:
Need to include the following:
"launch_language_code": "form_language_launch"
"submission_language_code": "form_language_submitted"

Reply:
added